inference-engine/include/ie_transformations.hpp

```diff
@@ -0,0 +1,54 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/**
+ * @brief This header file defines the list of public transformations.
+ *
+ * @file ie_transformations.hpp
```

Comment:
we need to add this header to `inference_engine.hpp`

Reply:
added